### PR TITLE
bump bundle to support Dropwizard 2.x.x

### PR DESCRIPTION
**Pull request by richeyh**
as 2.x.x has now been released anyone utilizing this bundle is going to need this bump in order to get there application working

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
   <groupId>org.pac4j</groupId>
   <artifactId>dropwizard-pac4j</artifactId>
   <packaging>jar</packaging>
-  <version>3.0.1-SNAPSHOT</version>
+  <version>4.0.0-SNAPSHOT</version>
```
Comment (richeyh) [inline pom.xml]: went ahead and bumped a major for this

**Comment by richeyh** [comment]
looks like a bunch of extraneous warn logging is coming in related to 
MultiException stack 1 of 3
org.glassfish.hk2.api.UnsatisfiedDependencyException: There was no object available for injection at SystemInjecteeImpl(requiredType=Pac4JValueFactoryProvider$Pac4JProfileValueFactoryProvider,parent=Pac4JValueFactoryProvider$ProfileInjectionResolver,qualifiers={},position=0,optional=false,self=false,unqualified=null,2086477093)
	at org.jvnet.hk2.internal.ThreeThirtyResolver.resolve(ThreeThirtyResolver.java:51)
	at org.jvnet.hk2.internal.ClazzCreator.resolve(ClazzCreator.java:188)
	at org.jvnet.hk2.internal.ClazzCreator.resolveAllDependencies(ClazzCreator.java:205)


after bumping the versions, i can either drop warn logs in the build or a fix will need to be put in upstream it looks like

**Comment by Naveg** [comment]
Looks like https://github.com/pac4j/jax-rs-pac4j/issues/45 was released. Can we move forward with this one now?

**Comment by Naveg** [comment]
@leleuj anything I can do to help with this one?

**Comment by leleuj** [comment]
Sure.

I plan to upgrade the next major versions of `jax-rs-pac4j` and `dropwizard-pac4j` to `pac4j` v4, but it won't be until September/October.

So we could create a version 4.0.0 of `dropwizard-pac4j` based on `jax-rs-pac4j` v3.0.3 and `Dropwizard` v2.
If you can submit a PR to do that (after the appropriate testing), I'll release the v4.0.0 of `dropwizard-pac4j` based on `Dropwizard` v2.

**Comment by leleuj** [comment]
I just upgraded `dropwizard-pac4j` v4.0.0-SNAPSHOT to pac4j v4, based on `jersey225-pac4j` v4.0.0-SNAPSHOT.
I'll first cut the v4.0.0 releases then we'll upgrade to Dropwizard v2.

**Comment by leleuj** [comment]
@richeyh Can you update this PR with version 5.0.0-SNAPSHOT for `dropwizard-pac4j` and Dropwizard v2?
Can you open another PR for the demo: https://github.com/pac4j/dropwizard-pac4j-demo? Thanks

**Comment by richeyh** [comment]
@leleuj pr has been update

**Comment by richeyh** [comment]
not sure whats going on with the build 
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-resources-plugin:2.6:resources (default-resources) on project dropwizard-pac4j: Execution default-resources of goal org.apache.maven.plugins:maven-resources-plugin:2.6:resources failed: Plugin org.apache.maven.plugins:maven-resources-plugin:2.6 or one of its dependencies could not be resolved: Failed to collect dependencies at org.apache.maven.plugins:maven-resources-plugin:jar:2.6 -> org.apache.maven:maven-plugin-api:jar:2.0.6: Failed to read artifact descriptor for org.apache.maven:maven-plugin-api:jar:2.0.6: Could not transfer artifact org.apache.maven:maven-plugin-api:pom:2.0.6 from/to central (https://repo.maven.apache.org/maven2): Connection reset -> [Help 1]

i didn't touch this so not sure whats wrong with travis trying to download this

**Comment by leleuj** [comment]
OK. Thanks for the update.

I restarted the build, Travis does not work very well lately.

Is JDK 11 mandatory for Dropwizard v2?

I created a 4.0.x branch and updated the master to v5.0.0-SNAPSHOT. This PR should go on a new major version.

Another PR must be opened for the demo as well: https://github.com/pac4j/dropwizard-pac4j-demo

**Comment by richeyh** [comment]
yes jdk11 is mandatory, ill try to get some time to rebase this then for the new master branch and get a PR open for that demo application

**Comment by richeyh** [comment]
PR is open for the demo dependent on this being completed

File: pom.xml [hunk 2]
```diff
@@ -90,7 +90,7 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <maven.compiler.source>1.8</maven.compiler.source>
```
Comment (leleuj) [inline pom.xml:91]: If JDK 11 is mandatory, this should be changed.

**Review by leleuj — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 3]
```diff
@@ -188,24 +315,17 @@
         <version>3.2.0</version>
       </plugin>
       <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>findbugs-maven-plugin</artifactId>
```
Comment (richeyh) [inline pom.xml:192]: find bugs is no longer supported and moved to spot bugs for java 11 support

**Comment by leleuj** [comment]
Thanks for your contribution. Merging

**Pull request merged**